numbers.csv

```diff
@@ -4,3 +4,4 @@ Sue,42
 Diana,27
 Jason,9
 Sophia, 10
+Daniel, 2
```

Comment:
Does emacs have a setting where newlines at the end of files are added automatically? Other text editors handle this by default.